src/System.Windows.Forms/src/System/Windows/Forms/MonthCalendar.CalendarButtonAccessibleObject.cs

```diff
@@ -2,51 +2,135 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Drawing;
+using System.Runtime.InteropServices;
 using static Interop;
 
 namespace System.Windows.Forms
 {
     public partial class MonthCalendar
     {
-        internal abstract class CalendarButtonAccessibleObject : CalendarChildAccessibleObject
+        /// <summary>
+        ///  Represents a base button accessible object for buttons in a MonthCalendar control.
```

Comment:
```suggestion
        ///  Represents an accessible object for buttons in <see cref="MonthCalendar"/> control.
```

Ditto in other places